README.md

```diff
@@ -147,61 +164,35 @@ containerize.  It is recommended that most users use a version tag (e.g.
 |`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
 
 See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
-Hub for a list of all the supported tags.
+Hub for a list of all the supported tags. -->
 
 ## Volumes ##
 
 | Mount point | Purpose        |
 |-------------|----------------|
-| `/var/log`  |  Log storage   |
-
-## Ports ##
-
-The following ports are exposed by this container:
-
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
```

Comment:
If no ports are being exposed, this section should look like this (just comment out the "ports" table in case it is ever needed in the future):

```
There are no ports exposed by this container.

<!--
The following ports are exposed by this container:

| Port | Purpose        |
|------|----------------|
| 8080 | Example only; nothing is actually listening on the port |

The sample [Docker composition](docker-compose.yml) publishes the
exposed port at 8080.
-->